packages/date-picker/src/FilterDateRangePicker/components/Trigger/FilterButtonGroup/FilterButtonGroup.module.scss

```diff
@@ -0,0 +1,84 @@
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/color";
+@import "../variables";
+
```

Comment:
This whole file is like witchcraft haha

Reply:
That's essentially what scss is. Precisely why our product teams don't like styling :P